private/pkg/manifest/module.go

```diff
@@ -34,3 +36,16 @@ func NewDigestFromBlobHash(hash *modulev1alpha1.Hash) (*Digest, error) {
 	}
 	return NewDigestFromBytes(dType, hash.Digest)
 }
+
+// NewBlobFromBytes creates a module Blob from in-memory content.
+func NewBlobFromBytes(content []byte) *modulev1alpha1.Blob {
+	blob := &modulev1alpha1.Blob{
+		Hash: &modulev1alpha1.Hash{
+			Kind:   modulev1alpha1.HashKind_HASH_KIND_SHAKE256,
+			Digest: make([]byte, 64),
+		},
+		Content: content,
+	}
+	sha3.ShakeSum256(blob.Hash.Digest, content)
```

Comment:
Why not using the digester interface? We could just receive the Digest+Content and optionally validate?

Reply:
We can now! I wrote this before but I missed the new stuff when rebasing. I'll clean this up. :)